src/pkg/runner/runner.go

```diff
@@ -467,7 +469,7 @@ func (r *Runner) checkForTaskLoops(task types.Task, tasksFile types.TasksFile, s
 		if action.TaskReference != "" {
 			exists := r.TaskNameMap[action.TaskReference]
 			if exists {
-				return fmt.Errorf("task loop detected")
+				return fmt.Errorf("task loop detected, ensure no cyclic loops in tasks or includes files")
```

Comment:
could show `action.TaskReference` in the error

Reply:
discussed IRL don't love it, but good enough for now considering our includes and task loop detection implementation. Will need to revisit how its being handled.